src/assets/i18n/en.json

```diff
@@ -54,6 +54,55 @@
       "3": "."
     }
   },
+  "address-book": {
+    "a-notification-will-be-shown-whenever-the-address-has-a-new": "A notification will be shown whenever the address has a new incoming transaction, sends or receives funds, or changes its representative.",
+    "account-address-copied-to-clipboard": "Account address copied to clipboard!",
+    "account-and-name-are-required": "Account and name are required",
+    "account-id-is-not-a-valid-account": "Account ID is not a valid account",
+    "add-new-entry": "Add New Entry",
+    "address-book-entry-saved-successfully": "Address book entry saved successfully!",
+    "address-book-export": "Address Book Export",
+    "address-book-export-downloaded": "Address book export downloaded!",
+    "address-book-import": "Address Book Import",
+    "address-book-too-large-for-a-qr-code": "Address book too large for a QR code.",
+    "back-to-address-book": "Back to address book",
+    "bad-import-data-make-sure-you-selected-a-nault-address-book": "Bad import data, make sure you selected a Nault Address Book export",
+    "edit-entry": "Edit Entry",
+    "exchange-deposit-address-main-trading-account-etc": "Exchange Deposit Address, Main Trading Account, etc.",
+    "export-address-book": "Export Address Book",
+    "export-all-address-book-entries-to-a-file-or-qr-code-url": "Export all address book entries to a file or QR Code/URL.",
+    "export-as-file": "Export As File",
+    "if-you-need-a-qr-use-fewer-entries-or-shorter-aliases": "If you need a QR, use fewer entries or shorter aliases.",
+    "import-address-book": "Import Address Book",
+    "import-address-book-entries-from-a-file": "Import address book entries from a file.",
+    "import-export": "Import / Export",
+    "note-exported-address-book-will-not-be-encrypted-by-your-wal": "Note: Exported address book will NOT be encrypted by your wallet password.",
+    "reload-balances": "Reload balances",
+    "scan-the-qr-code-on-any-device-to-import-your-address-book": "Scan the QR code on any device to import your address book!",
+    "show-total-tracked-balance": "Show Total Tracked Balance",
+    "successfully-deleted-address-book-entry": "Successfully deleted address book entry",
+    "the-balance-will-be-displayed-in-the-address-book": "The balance will be displayed in the address book.",
+    "this-name-is-already-in-use-please-use-a-unique-name": "This name is already in use! Please use a unique name",
```

Comment:
could be reused here

https://github.com/Nault/Nault/blob/ac33875a2773a06c5e0cc341a84aa2520e5033d3/src/app/components/account-details/account-details.component.ts#L563